test/runnable/imports/bar10378.d

```diff
@@ -0,0 +1,4 @@
+
+module bar;
```

Comment:
A `module bar;` is imported by `import imports.bar10378;`. It's a mismatch.

I'm not sure why the error is not reported, but at least, this line should be: `module imports.bar10378;`.